backend/packages/Upgrade/src/api/services/MoocletExperimentService.ts

```diff
@@ -136,6 +136,45 @@ export class MoocletExperimentService extends ExperimentService {
     return this.dataSource.transaction((manager) => this.handleDeleteMoocletTransaction(manager, params));
   }
 
+  public async syncImportExperiment(
+    experimentFiles: ExperimentFile[],
+    user: UserDTO,
+    logger: UpgradeLogger
+  ): Promise<ValidatedExperimentError[]> {
+    const { experiments, validatedExperiments } = await this.verifyExperiments(experimentFiles, logger);
+
+    await this.syncAddBulkExperiments(experiments, user, logger);
+    return validatedExperiments;
+  }
+
+  private async syncAddBulkExperiments(
+    experiments: ExperimentDTO[],
+    currentUser: UserDTO,
+    logger: UpgradeLogger
+  ): Promise<Experiment[]> {
+    const createdExperiments = [];
+    for (const experiment of experiments) {
```

Comment:
Can `for` loop be replaced with `map` so the creation will be parallel?

Reply:
Probably, I'll try